packages/stream-tests/src/__tests__/deterministic-indexing.test.ts

```diff
@@ -0,0 +1,110 @@
+import { jest } from '@jest/globals'
```

Comment:
there is already a `deterministic-model-instance-document.test.ts` file in the `stream-tests` package.  Should we add these test cases into that file instead of making a new test file?

Reply:
The added tests are more specific to the behavior or deterministic MIDs with indexing so I found it more convenient to have a dedicated test suite, but I can move the tests to the other suite if you prefer?

Reply:
I don't feel strongly.  If you think it makes sense to keep them separate that's fine